specs/pvs/corla.pvs

```diff
@@ -541,16 +586,27 @@ END ballots_collections
 
 instructions_forms_reports: THEORY
 BEGIN
-  IMPORTING information_systems
+  IMPORTING information_systems, ballot_manifests, election, audits
   
-  report: TYPE
+  report: TYPE = [# name: string #]
 % @review CDOS The contents of this report are underspecified at the
 % moment. See submit_audit_mistake_report below for more information.
-  audit_mistake_report: TYPE FROM report
+  audit_mistake_report: TYPE =
```

Comment:
I suggest we change the term "mistake" to "investigation".

If the audit board enters an interpretation of a sampled ballot which is discrepant with the CVR, there are a number of possibilities to investigate. Some represent discrepancies to be handled via the RLA math, and others might simply indicate that the wrong ballot was retrieved, and they may be looking at the wrong paper ballot. I'll flesh these out.